===================================================================
RCS file: /cvs/glibc/libc/nptl/sysdeps/unix/sysv/linux/x86_64/sem_trywait.S,v
retrieving revision 1.6
retrieving revision 1.6.4.1
diff -u -r1.6 -r1.6.4.1
--- libc/nptl/sysdeps/unix/sysv/linux/x86_64/sem_trywait.S	2003/05/10 20:37:38	1.6
+++ libc/nptl/sysdeps/unix/sysv/linux/x86_64/sem_trywait.S	2005/10/24 20:50:54	1.6.4.1
@@ -1,4 +1,4 @@
-/* Copyright (C) 2002, 2003 Free Software Foundation, Inc.
+/* Copyright (C) 2002, 2003, 2005 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Ulrich Drepper <drepper@redhat.com>, 2002.
 
@@ -56,4 +56,3 @@
 	orl	$-1, %eax
 	retq
 	.size	sem_trywait,.-sem_trywait
-	versioned_symbol(libpthread, __new_sem_trywait, sem_trywait, GLIBC_2_1)
